buster-amd64/bin/dist_iso

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
```

Comment:
/bin/sh isn't enough?

Reply:
/bin/sh doesn't support echo -e "\n". Changed these to additional echoes.